packages/gatsby-transformer-excel/README.md

```diff
@@ -63,11 +63,11 @@ So if your project has a `letters.xlsx` with two worksheets:
 the following nodes would be created:
 
 ```javascript
```

Comment:
please change `javascript` here to `json5`, otherwise prettier will format this again to what it was before

Reply:
ok, all set @pieh